assets/js/translator.js

```diff
@@ -740,23 +739,33 @@ function translateDoc() {
                         });
                     }
                     else if(this.readyState === XHR_DONE && xhr.status === HTTP_OK_CODE) {
-                        downloadBrowserWarn();
                         $('div#fileUploadProgress').parent().fadeOut('fast');
                         $('div#fileLoading').fadeOut('fast', function () {
                             if(typeof window.navigator.msSaveBlob !== 'undefined') {
+                                // IE file download workaround
                                 $('a#fileDownload')
                                     .click(function () {
                                         window.navigator.msSaveBlob(xhr.response, fileName);
-                                    })
-                                    .fadeIn('fast');
+                                    });
                             }
-                            else {
-                                var URL = window.URL || window.webkitURL;
-                                $('a#fileDownload')
-                                    .attr('href', URL.createObjectURL(xhr.response))
-                                    .attr('download', fileName)
-                                    .fadeIn('fast');
+                            else if(/.*(?!chrome|android).*(version\/\d\.|Mobile).*safari/i.test(window.navigator.userAgent) &&
```

Comment:
A little explanation for this, perhaps?

Reply:
@sushain97 we can do this ourselves really, after merging.

Reply:
Tests User Agent against a regexp to detect if the user is on older or mobile version of Safari and then runs another test to eliminate Chrome, which also include Safari in User Agent.

Should I add there a comment? I thought that the one line below is enough

Reply:
Yes, just expand that comment a bit. The thing you just said would be perfect.